azext_iot/central/_help.py

```diff
@@ -136,6 +136,51 @@ def _load_central_devices_help():
                 az iot central app device registration-summary --app-id {appid}
         """
 
+    helps[
+        "iot central app device run-command"
```

Comment:
Is "run command" IoT Central nomenclature? On the IoT hub side its usually invoke or execute command.

Reply:
I'll check with my team about "invoke". My team was trying to avoid "execute"

Reply:
So I checked with the team, they want to keep Run because "run" is what our UI says

![image](https://user-images.githubusercontent.com/42591144/86292379-7da22e00-bba5-11ea-8612-db259b139b43.png)

Reply:
#wontfix for now